core/src/main/java/io/grpc/Status.java

```diff
@@ -383,7 +414,7 @@ public Throwable getCause() {
   }
 
   /**
-   * Is this status OK, i.e. not an error.
+   * Is this status OK, i.e., not an error.
```

Comment:
Without the comma the one sentence description is "Is this status OK, i.e.". The comma solves the problem and is still grammatically correct, so I went with it.